code/game/objects/structures/cas_plane_parts.dm

```diff
@@ -145,6 +145,13 @@
 	///ref to the static weapon
 	var/obj/structure/dropship_equipment/weapon/static_weapon
 
+/obj/structure/caspart/minigun/examine(mob/user)
+	. = ..()
+	if(static_weapon.ammo_equipped)
+		static_weapon.ammo_equipped.show_loaded_desc(user)
```

Comment:
make an early return so you can remove the else, or even better, inline this

Reply:
Not sure what this means, sorry.

Reply:
usually not worth the effort to change, but bravemole would be correct here
```
//They mean this
if(!x)
   return
if(!y)
  return
stuff()
//as opposed to 
if(x)
   if(y)
       stuff()

```